.github/workflows/publish-docusaurus.yaml

```diff
@@ -0,0 +1,115 @@
+#
+#  Copyright (c) 2023 Mercedes Benz Tech Innovation GmbH
+#  Copyright (c) 2021, 2023 Contributors to the Eclipse Foundation
+#
+#  See the NOTICE file(s) distributed with this work for additional
+#  information regarding copyright ownership.
+#
+#  This program and the accompanying materials are made available under the
+#  terms of the Apache License, Version 2.0 which is available at
+#  https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations
+#  under the License.
+#
+#  SPDX-License-Identifier: Apache-2.0
+#
+
+---
+name: "Publish docusaurus documentation"
+
+on:
+  workflow_dispatch:
+    inputs:
+      downstream-version:
+        required: false
+        type: string
+  workflow_call:
+    inputs:
+      downstream-version:
+        required: false
+        type: string
+
+jobs:
+  publish-docusaurus:
+    permissions:
+      contents: write
+      packages: write
+    name: "Publish docusaurus documentation as zip file"
+    runs-on: ubuntu-latest
+    env:
+      SWAGGERHUB_API_KEY: ${{ secrets.SWAGGERHUB_API_KEY }}
+      SWAGGERHUB_USER: ${{ secrets.SWAGGERHUB_USER }}
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: "Setup node"
+        uses: actions/setup-node@v3
+
+      - name: "Install tools"
+        run: |
+          npm i -g swaggerhub-cli markdownlint-cli2 @synx-ai/oas3-mdx
+
+      - name: "Extract version"
+        id: extract_version
+        run: |
+          if [ -z ${{ inputs.downstream-version }} ]; then
+            echo "DOWNSTREAM_VERSION=$(sed -nr "{ :l /^version[ ]*=/ { s/[^=]*=[ ]*//; p; q;}; n; b l;}" ./gradle.properties | tr -d ' ' | tr -d '"')" >> "$GITHUB_OUTPUT"
```

Comment:
please use the commands from the `publish-swaggerhub.yaml` file.

Reply:
I think there's a problem with those. I've run into a case where the timing of the substitution of variables by the workflow led to an empty variable being used. Hence I switched to this approach and would recommend adapting it for `publish-swaggerhub.yaml` too.